lit/SwiftExpr/Dict.test

```diff
@@ -0,0 +1,50 @@
+# Test that dictionaries work in the REPL.
+# REQUIRES: darwin
+
+# RUN: %lldb --repl < %s | FileCheck %s --check-prefix=DICT
+
+[1:2, 2:3, 3:9]
+# DICT: Welcome to Swift version {{.*}} (LLVM {{.*}}, Clang {{.*}}, Swift {{.*}})
+# DICT: $R0: [Int : Int] = 3 key/value pairs {
+# DICT:   [0] = {
```

Comment:
Now that we're using FileCheck, we can tighten these checks up with -NEXT. But it'll do as a follow-up.